src/TodoContext/TodoContext.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
If I remove this, I get errors from lines like this:
`setError: (_value: ErrorMessages | null) => {}`
and because of this I can't type context properly